### DISCUSSION TITLE
open API

### DISCUSSION THREAD

**Discussion by ericwu54** [consenlabs/webview#49] [Q&A] [answered]
想請問一下有沒有getQuote()的API提供

**Comment by izayl** [comment, marked as answer]
老版本的 API：https://docs.token.im/tokenlon-jssdk/en/#getquote   目前已经停止维护了

最新的 API 会在近期公开，请关注 https://twitter.com/tokenlon 获得第一时间的更新

**Reply by ericwu54** [reply]
感謝 
這個他說要upgrade是因為沒在maintain了嗎, 還是其他地方有錯
[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "tokenlon_updated_you_need_upgrade".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
